README.md

```diff
@@ -32,24 +32,25 @@ $ sudo apt install nicotine
 Package maintainers, please insert instructions for users to install pre-compiled packages from your respective repositories here.
 
 ## Windows
-- [Nicotine+ Installer](https://github.com/Nicotine-Plus/nicotine-plus/releases/download/2.0.0/Nicotine+-2.0.0.exe)  
+- [Nicotine+ Installer](https://github.com/Nicotine-Plus/nicotine-plus/releases/download/2.0.0/Nicotine+-2.0.0.exe)
 
-# License
-
-Nicotine+ is released under the terms of the [GNU Public License v3](https://www.gnu.org/licenses/gpl-3.0-standalone.html) or later.
+## Building from git
+Building from git is described in [RUNFROMGIT.md](doc/RUNFROMGIT.md)
```

Comment:
Add a sentence or two about this only being encouraged for testers, e.g. https://github.com/Nicotine-Plus/nicotine-plus/pull/392/files

Reply:
Yes, good idea. I've added that to the RUNFROMGIT.md. Now I have to figure out if the change made it into the pull request.

Reply:
Please check https://github.com/hboetes/nicotine-plus/blob/master/README.md to get an overview on the whole README. I put the build from git on the logical spot, as the last of the installation options, and the licence next to other legal stuff.
I hope that satisfies your request.